public/static/docs/command-reference/remote/modify.md

```diff
@@ -403,6 +403,69 @@ including obtaining the necessary credentials, and how to form `gdrive://` URLs.
 
 </details>
 
+<details>
+
+### Click for HTTP
+
+- `auth` - authentication method to use when accessing a remote. Accepted values
+  are:
+
+  - `basic` - use HTTP basic access authentication (`user` and `password`
+    options should also be provided).
```

Comment:
``[Basic authentication scheme](https://tools.ietf.org/html/rfc7617). `user` and `password` (or `ask_password`) parameters should also be configured.``